src/windows/custom_currency.c

```diff
@@ -0,0 +1,287 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "../localisation/localisation.h"
+#include "../interface/widget.h"
+#include "../interface/window.h"
+#include "../config.h"
+#include "dropdown.h"
+
+enum WINDOW_CUSTOM_CURRENCY_WIDGET_IDX {
+    WIDX_BACKGROUND,
```

Comment:
Following coding standards _is_ a priority in my book, but the wiki page starts off with saying it isn't. But de facto they are, we have frequently asked people to adhere. Clearly this needs discussion about those and about their enforcement, but not in this PR. 

Follow the standards in that page, but use tabs to indent (with a width of 4 spaces) for consistency with other C files.